both/_round.js

```diff
@@ -0,0 +1,175 @@
+
+// Add functions to the Mongo object, using transform (see http://docs.meteor.com/#/full/mongo_collection)
+Round = function(doc) {
+  _.extend(this, doc);
+};
+_.extend(Round.prototype, {
+
+  format: function() {
+    return wca.formatByCode[this.formatCode];
+  },
+  properties: function() {
```

Comment:
I just couldn't have "round.round" in the code, so came up with this bland vague name.

I think it's hard to name these things because they should really be individual properties or functions on the Round class itself. "code" is already there, and the others look quite computable.